package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/icons": "^1.0.13",
```

Comment:
Can we reuse react-icons here? If not, then can we switch to only one icon library for consistency?

Reply:
Shall we use only chakra-ui/icons and remove react-icons from dependencies? or the other way round? what are your thoughts, I personally was thinking to maybe use chakra icons as we don't need large amount of icons provided by react-icons for now